services/validation.js

```diff
@@ -0,0 +1,9 @@
+const validUrl = require("valid-url");
+
+const isValidUrl = (url) => {
+  // If the URL is valid, isUri returns the URL
+  // If the URL is not valid, isUri returns undefined
+  return validUrl.isUri(url);
```

Comment:
Currently, the function returns the URL if it's valid and undefined if it's not. It would be best for it to return a boolean.

It should return this  `return !!validUrl.isUri(url);` of which the return statement will other be true or false